docs/io/index_unofficial.rst

```diff
@@ -0,0 +1,287 @@
+container for that data format. This container must contain the same
+information as `ctapipe.io.containers.R0CameraContainer`, but can contain
+additional information which may be used by the
+`ctapipe.calib.camera.r1.CameraR1Calibrator`. An example of this for
+the TargetioEventReader is the `first_cell_ids` field in
+`ctapipe.io.unofficial.targetio.containers.TargetioR0CameraContainer`.
+
+Additionally, to automate the selection of the correct
+`ctapipe.calib.camera.r1.CameraR1Calibrator` from
+`ctapipe.calib.camera.r1.CameraR1CalibratorFactory`, the
+`ctapipe.io.eventfilereader.EventFileReader.r1_calibrator` property can be
+used to specify the correct calibrator to use.
+This is then obtained from the reader, and passed to the
+`ctapipe.calib.camera.r1.CameraR1CalibratorFactory`. An example of this can be
+found in ctapipe/examples/calibration_pipeline.py. If a particular data format
+is to be used by multiple cameras, each requiring different R1 calibration,
+then this method should correctly return the right one. By default, a new
+reader will return 'NullR1Calibrator'.
+
+
+Instrument Configuration
+========================
+
+As the `ctapipe.instrument` module and "instrument database" are not fully
+developed, the camera configuration information (such as pixel positions) are
+still usually obtained from inside the hessio file.
+
+Data formats other than hessio are unlikely to contain the camera
+configuration, therefore until the `ctapipe.instrument` module and the
+"instrument database" are completed, the unofficial readers must contain other
+methods to load the instrument information into the container. This can
+involve loading data from additional files, as long as these data files are
+added to the `data_files` dictionary in setup.py.
+
+For an example of one approach to achieve this, see
+`ctapipe.io.unofficial.targetio.camera`.
+
+
+Tests and Documentation
+=======================
+
+Every unofficial reader must be thoroughly tested and documented in their
+docstrings. This is especially crucial for these modules due to their
+uniqueness and differences between one reader and another.
+
+As we want to avoid adding new depencies to ctapipe, one should make tests of
+their EventFileReader skippable if the external dependencies do not exist on
+the machine. This can be done by either:
+
+>>> import pytest
+>>> pytest.importorskip("external_module")
+
+To skip all the tests in the file, or:
+
+>>> import pytest
+>>> def test_something():
+>>>    pytest.importorskip("external_module")
+
+To skip a single test function.
+
+The method to install the external software should be added to the .travis.yml
+file, so that the Travis CI can perform all tests, including the unofficial
+reader tests. Additionally, a small test file that is in an unofficial data
+format should be committed to ctapipe-extra.
+
+Installation instructions for external software should be included in the
+docstrings of the EventFileReader.
+
+
+TargetioFileReader
```

Comment:
I don't this this should reside here. This looks like it belongs into the class docstring of TargetioFileReader.